app/views/service_providers/_form.html.erb

```diff
@@ -112,10 +112,67 @@
                    hint: 'The name of the <a href="https://github.com/18F/identity-idp/tree/main/app/assets/images/sp-logos" target="_blank">logo image file</a> in the IdP. The login.gov team can add this image for you.'.html_safe %>
   <% end %>
 
-  <%= form.input :saml_client_cert,
-                 label: 'Public certificate',
-                 input_html: { class: 'usa-textarea' },
-                 hint: 'Your public certificate, which contains your public key, is needed for OpenID Connect (when using <i>private_key_jwt</i>) and for SAML. This must be <a href="https://en.wikipedia.org/wiki/Privacy-enhanced_Electronic_Mail" target="_blank">PEM encoded</a>, for example in the form: <pre><code>-----BEGIN CERTIFICATE-----<br>MIIDXTCCAkWgAwIBAgIJAJC1HiIAZAiIMA0GCSqGSIb3Df<br>BAYTAkFVMRMwEQYDVQQIDApTb21lLVN0YXRlMSEwHwYDVx<br>B7xxt8BVc69rHV15A0qyx77CLSj3tCx2IUXVqRs5mlvA==<br>-----END CERTIFICATE-----</code></pre>'.html_safe %>
+  <fieldset class="usa-fieldset">
+    <%= form.label 'Certificates', for: :saml_client_cert, class: 'usa-label' %>
```

Comment:
This doesn't feel like a label anymore, since `saml_client_cert` input doesn't exist. Maybe `legend` could work, though if we're just going for the visual effect, a `div` could be fine to?

```suggestion
    <div class="usa-label">Certificates</div>
```